From 1c9c269fca3b1dc0532af45d8b15ec2e64e35a06 Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Sat, 15 Sep 2018 15:51:02 +0800
Subject: [PATCH 4719/5242] MLK-19664-4 ARM64: dts: imx8qm domu: added phandle
 and linux,phandle for gic node

commit  0a4da1a15aa867e9cf21591eb9d177bf4905c519 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add phandle and linux,phandle for gic node.
When booting Domu with device tree, xen tool will create gic node
with phandle PHANDLE_GIC(65000), so in domu dts, we need to code
that in gic node, make sure it is same as what xen tool use.
Otherwise, wrong gic phandle might be used.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-domu.dts     |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
index e6b0bf8..2e0080c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
@@ -118,6 +118,8 @@
 		      <0x0 0x3020000 0 0x1000000>; /* GICR */
 		interrupts = <GIC_PPI 9 (GIC_CPU_MASK_SIMPLE(6) | IRQ_TYPE_LEVEL_HIGH)>;
 		interrupt-parent = <&gic>;
+		linux,phandle = <0xfde8>;
+		phandle = <0xfde8>;
 	};
 
 	timer {
-- 
1.7.9.5

